crates/bitwarden-uniffi/src/lib.rs

```diff
@@ -137,24 +141,27 @@ fn init_logger() {
             .with_target(true)
             .pretty();
 
+        // Build base registry once instead of duplicating per-platform
+        let registry = tracing_subscriber::registry().with(fmtlayer).with(filter);
+
+        // Conditionally add callback layer if provided
+        // Use Option to avoid type incompatibility between Some/None branches
+        let callback_layer = callback.map(log_callback::CallbackLayer::new);
+        let registry = registry.with(callback_layer);
```

Comment:
Do we want the callback layer to have the same terminal-focused formatting the registry applies? I would think we'd just want plain text for these logs, since we don't know where they are going to end up displayed.